cleo_sdk/CLEO_Utils.h

```diff
@@ -418,6 +420,9 @@ namespace CLEO
     #define OPCODE_READ_PARAM_FLOAT() _readParamFloat(thread).fParam; \
         if (!IsLegacyScript(thread) && !_paramWasFloat()) { SHOW_ERROR("Input argument #%d expected to be float, got %s in script %s\nScript suspended.", CLEO_GetParamsHandledCount(), CLEO::ToKindStr(_lastParamType, _lastParamArrayType), CLEO::ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
 
+    #define OPCODE_READ_PARAM_ANY32() _readParam(thread).dwParam; \
+        if (!_paramWasInt() && !_paramWasFloat()) { SHOW_ERROR("Input argument #%d expected to be simple-type, got %s in script %s\nScript suspended.", CLEO_GetParamsHandledCount(), CLEO::ToKindStr(_lastParamType, _lastParamArrayType), CLEO::ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
```

Comment:
can we use explicit types instead of "simple-type". It's not obvious what simple-type is.

for example:
"Input argument #%d expected to be an int or float,

Reply:
Right. I fixed it only in one place earlier.